src/unix/aix.c

```diff
@@ -760,16 +760,15 @@ static void uv__ahafs_event(uv_loop_t* loop, uv__io_t* event_watch, unsigned int
       p++;
     }
   } else {
-    /* For file changes, figure out whether filename is absolute or not */
-    if (handle->path[0] == '/') {
-      p = strrchr(handle->path, '/');
+    p = strrchr(handle->path, '/');
+    if (p == NULL)
+      p = handle->path;
+    else
+      p++;
+    while (*p != NULL) {
+      fname[i]= *p;
```

Comment:
Well fname is initialized to MAX_PATH size and ```i```'s value is dependent on the file name pointed to by ```handle->path```

 To be honest we could just get rid off of ```i``` all together if we just used ```memset``` and ```strcpy``` instead. Would look a little cleaner imo.